### DISCUSSION TITLE
다음에는 작업 계획 세울 때 같이 해볼까요?

### DISCUSSION THREAD

**Discussion by jongfeel** [wisdom08/dev-community#31] [Ideas]
프로젝트 진행하는데 시간을 많이 못쓴다고 하셨긴 했는데
어쨌든 진행은 조금씩 하실 거고
다음 iteration 4도 그렇게 진행 예정이라면 다음과 같이 해봤으면 좋겠습니다.

- 작업 시간 산정, 시간 단위
- Iteration 4 작업 계획 목록 정하기, 날짜 단위
- 각 작업 계획 별 issue 생성 및 정리

다음 미팅 때 같이 해보고
어느 정도 범위에 대해 얘기할 댸 저도 범위 산정 하는데 가이드를 드려볼 예정입니다.

**Comment by wisdom08** [comment]
넵 말씀해주신 부분에 대해서 미리 생각해보고 미팅 참여하겠습니다~!
